docs/src/main/paradox/classic.md

```diff
@@ -0,0 +1,43 @@
+# Akka Classic
+
+Akka Projections can be used with the [new Actor API](https://doc.akka.io/docs/akka/current/typed/actors.html) or
+the [classic Actor API](https://doc.akka.io/docs/akka/current/index-classic.html). The documentation samples
+show the new Actor API, and this page highlights how to use it with the classic Actor API.
+
+## Actor System
+
+The `ActorSystem` is a parameter in several places of the Projections API. That is the `akka.actor.typed.ActorSystem`.
```

Comment:
Would it make sense to add apidocs to the classic and typed ActorSystems?

Reply:
The important part is to show the package name, which I don't think apidoc links show.

Reply:
I think you're right. :+1: